packages/angular-workspace/nimble-angular/label-provider/core/nimble-label-provider-core-with-defaults.directive.ts

```diff
@@ -1,7 +1,7 @@
 import { Directive, ElementRef, Renderer2 } from '@angular/core';
 import type { LabelProviderCore } from '@ni/nimble-components/dist/esm/label-provider/core';
 
-import '@angular/localize/init';
+declare const $localize: (messageParts: TemplateStringsArray) => string;
```

Comment:
There were warnings that you should not import @angular/localize/init. We just need a declaration to keep TS happy.

Reply:
Looking at the recent build: https://github.com/ni/nimble/actions/runs/20310817621/job/58341216982#step:15:428

I expect the current push to break in test as well. I think in angular.json in polyfills you need to add `@angular/localize/init`: https://stackoverflow.com/a/78636646